tests/data/specs/in_p1.rflx

```diff
@@ -0,0 +1,9 @@
+with P1;
+with P2;
+with P3;
+package In_P1 is
+   for P1::Frame use (Payload => P2::Packet)
+      if Kind = 1;
+   for P1::Frame use (Payload => P3::Packet)
+      if Kind = 2;
+end In_P1;
```

Comment:
What about adding blank lines? I would prefer to format test specs (especially the ones in separate files) in the same way as other example specs.

Reply:
Changed in 219cbbda.